apache2/waf_logging/waf_log_util_internal.h

```diff
@@ -1,9 +1,100 @@
 #ifndef _WAF_LOG_UTIL_INTERNAL_HEADER
 #define _WAF_LOG_UTIL_INTERNAL_HEADER
 
-#include <google/protobuf/util/json_util.h>
-#include"waf_format.pb.h"
 #include <fstream>
+#include <unordered_map>
 #include <exception>
+#include "../../jsoncons/include/jsoncons/json.hpp"
```

Comment:
This and next line - please refrain from using backward-relative paths containing "../".
Instead, make sure that the project's root directory is in the includes list so they can be tracked from there.

Reply:
since in the lib, the include path are <>, and we have to update the docker image, so i will change back to <>